packages/trader/src/Modules/SmartChart/index.tsx

```diff
@@ -0,0 +1,258 @@
+import React from 'react';
+import { getUrlBase, moduleLoader } from '@deriv/shared';
+import { useStore } from '@deriv/stores';
+import {
+    ActiveSymbols,
+    ActiveSymbolsRequest,
+    AuditDetailsForExpiredContract,
+    ProposalOpenContract,
+    ServerTimeRequest,
+    TickSpotData,
+    TicksHistoryRequest,
+    TicksHistoryResponse,
+    TicksStreamResponse,
+    TradingTimes,
+    TradingTimesRequest,
+} from '@deriv/api-types';
+import { TBottomWidgetsParams } from 'Modules/Trading/Containers/trade';
+import type { TChartLayout, TChartStateChangeOption } from 'Stores/Modules/Trading/trade-store';
+
+type TMarkersArray = ReturnType<typeof useStore>['contract_trade']['markers_array'];
+type TGetContractsArray = ReturnType<typeof useStore>['contract_replay']['contract_store']['getContractsArray'];
+type TBarriers = ReturnType<typeof useStore>['portfolio']['barriers'];
+
+type TContractsArray = TMarkersArray | ReturnType<TGetContractsArray>;
+
+type TGetIndicatorHeightRatio = (chart_height: number, indicator_count: number) => TRatio;
+
+type TInitialChartData = {
+    activeSymbols?: ActiveSymbols;
+    masterData?: TQuote[];
+    tradingTimes?: TradingTimes;
+};
+
+type TLanguage = {
+    icon: JSX.Element;
+    key: string;
+    name: string;
+};
+
+type TNotification = {
+    category: string;
+    text: string;
+    type?: 'info' | 'warning' | 'success' | 'error' | 'warning';
```

Comment:
extra 'warning'

Reply:
I will merge this PR into the parent branch and will remove extra 'warning'

Reply:
thank you @kate-deriv :)